aspnetcore/tutorials/first-mvc-app/adding-controller.md

```diff
@@ -60,69 +89,99 @@ Name the controller **HelloWorldController**.
 
 Replace the contents of *Controllers/HelloWorldController.cs* with the following:
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie/Controllers/HelloWorldController.cs?name=snippet_1)]
+  [!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie/Controllers/HelloWorldController.cs?name=snippet_1)]
 
 Every `public` method in a controller is callable as an HTTP endpoint. In the sample above, both methods return a string. Note the comments preceding each method.
 
-An HTTP endpoint is a targetable URL in the web application, such as `https://localhost:5001/HelloWorld`, and combines the protocol used: `HTTPS`, the network location of the web server (including the TCP port): `localhost:5001` and the target URI `HelloWorld`.
+An HTTP endpoint:
 
-The first comment states this is an [HTTP GET](https://www.w3schools.com/tags/ref_httpmethods.asp) method that's invoked by appending `/HelloWorld/` to the base URL. The second comment specifies an [HTTP GET](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html) method that's invoked by appending `/HelloWorld/Welcome/` to the URL. Later on in the tutorial the scaffolding engine is used to generate `HTTP POST` methods which update data.
+* Is a targetable URL in the web application, such as `https://localhost:5001/HelloWorld`.
+* Combines:
 
-Run the app in non-debug mode and append "HelloWorld" to the path in the address bar. The `Index` method returns a string.
+  * The protocol used: `HTTPS`.
+  * The network location of the web server (including the TCP port): `localhost:5001`.
+  * The target URI: `HelloWorld`.
 
-![Browser window showing an application response of This is my default action](~/tutorials/first-mvc-app/adding-controller/_static/hell1.png)
+The first comment states this is an [HTTP GET](https://www.w3schools.com/tags/ref_httpmethods.asp) method that's invoked by appending `/HelloWorld/` to the base URL.
```

Comment:
Use Monzilla, not w3schools
https://developer.mozilla.org/docs/Web/HTTP/Methods/GET